From 583223729834cfc179d2da3ef3932d3dc07b11dc Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Mikko=20Salom=C3=A4ki?= <ms@datarespons.se>
Date: Tue, 31 Mar 2020 07:52:26 +0200
Subject: [PATCH] vec6200: add dt to Makefile

---
 arch/arm/dts/Makefile | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/arch/arm/dts/Makefile b/arch/arm/dts/Makefile
index 0e2ffdb87f..b8bd08b0f2 100644
--- a/arch/arm/dts/Makefile
+++ b/arch/arm/dts/Makefile
@@ -513,7 +513,8 @@ dtb-$(CONFIG_MX6QDL) += \
 	imx6dl-sabreauto.dtb \
 	imx6dl-sabresd.dtb \
 	imx6qp-sabreauto.dtb \
-	imx6qp-sabresd.dtb
+	imx6qp-sabresd.dtb \
+	vec6200-dl-revA.dtb
 
 dtb-$(CONFIG_MX6SL) += imx6sl-evk.dtb
 
